.github/workflows/integration-test.yml

```diff
@@ -49,27 +50,6 @@ jobs:
           done
           docker exec ibm-mq-test dspmq
 
-      - name: Create test queues and channel
```

Comment:
so, now that we have data in the channel.  i guess these are not needed then.  
actually, they might not be needed to begin with.

Reply:
it's a start but it's not enough, we need to create channels and create connections to them. I am adding that in a separate PR.

Reply:
created https://splunk.atlassian.net/browse/POR-20491 to track this.